triplea_maps.yaml

```diff
@@ -2586,3 +2586,10 @@
   img: https://raw.githubusercontent.com/triplea-maps/TerraFirma1939/master/preview.png
   description: |
    <br>World War 2 on a huge map
+- mapName: WWI in 1939
```

Comment:
There is a good chance this will break as the map name must be very similar to the folder name & repo name. The repo name is 'wwi_1939', so the map could be called 'WWI 1939'

It is pretty easy to rename repo's. If you do that, take care to update the 'xml' map (and/or game name) tags in each XML too if doing so.